app/services/transaction_search/date_range_searcher.rb

```diff
@@ -25,9 +29,14 @@ def search(params)
       end_date = to_date(params[:end])
       @date_range_field = extract_date_range_field(params)
 
-      order_details
-        .action_in_date_range(@date_range_field, start_date, end_date)
-        .ordered_by_action_date(@date_range_field)
+      if start_date.present? || end_date.present?
+        order_details
+          .action_in_date_range(@date_range_field, start_date, end_date)
+          .ordered_by_action_date(@date_range_field)
```

Comment:
We shouldn't have the conditional around this. We _always_ want to search by the date range field. and `action_in_date_range` is smart enough to deal with `nil`s. What are you trying to accomplish with the conditional?

Reply:
Ah okay we can take this out actually. We were seeing no results again when the date filter was added, but I think it was because the field was set to "Fulfilled". Now that it defaults to "Ordered" it's returning results again.